servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcExecutionStrategies.java

```diff
@@ -25,10 +25,20 @@
 public final class GrpcExecutionStrategies {
 
     private static final GrpcExecutionStrategy NEVER_OFFLOAD_STRATEGY =
-            new DefaultGrpcExecutionStrategy(HttpExecutionStrategies.offloadNever());
+            new DefaultGrpcExecutionStrategy(HttpExecutionStrategies.offloadNever()) {
+                @Override
+                public HttpExecutionStrategy merge(final HttpExecutionStrategy other) {
+                    return this;
+                }
+            };
 
     private static final GrpcExecutionStrategy DEFAULT_GRPC_EXECUTION_STRATEGY =
-            new DefaultGrpcExecutionStrategy(HttpExecutionStrategies.defaultStrategy());
+            new DefaultGrpcExecutionStrategy(HttpExecutionStrategies.defaultStrategy()) {
+                @Override
+                public HttpExecutionStrategy merge(final HttpExecutionStrategy other) {
+                    return other;
```

Comment:
This is an unfortunate necessity currently. The alternative is to include explicit checks for this constant everywhere `HttpExecutionStrategies.defaultStrategy()` is being detected.

Given the ways in which `GrpcExecutionStrategy` is used the best option may be just to eliminate it entirely and use `HttpExecutionStrategy` everywhere.

Reply:
From what I remember, last time we discussed possibility to remove `GrpcExecutionStrategy` we decided to keep it because it may be useful in future to control offloading for grpc-specific features. For example, users may prefer to configure offloading only for serialization/deserialization or compression/decompression but not for the request/response processing. Having it just gives us more flexibility in future.

Reply:
Understood. For now the focus will be on removing the need for the special constants entirely and `GrpcExecutionStrategy` can continue to be used the same way it is currently.